tests/hopr-smoke-test-node_default.cfg.yaml

```diff
@@ -0,0 +1,16 @@
+---
+hopr:
+  strategy:
+    on_fail_continue: true
+    strategies:
+      - !AutoFunding
+        funding_amount: "10000000000000000000 HOPR"
+        min_stake_threshold: "1000000000000000000 HOPR"
+      - !Aggregating
+        aggregation_threshold: 100
```

Comment:
Don't we want to lower this to something like 20? Also the `TICKET_AGGREGATION_THRESHOLD` value in `conftest.py`

Reply:
Not necessary at this point.